### PR TITLE
test(annotation): validation function

### PR DESCRIPTION
**Pull request by MilanBogicevic8**

### CODE REVIEW

File: annotation/tests/test_tasks_services.py [hunk 1]
```diff
@@ -1746,3 +1746,120 @@ def test_get_tasks_without_ids_with_multiple_task_types():
...
     assert services.get_tasks_without_ids(tasks) == expected
+
+
+def test_create_validation_revisions_successful(
+    mock_session: Mock,
+    mock_get_file_path_and_bucket: Mock,
+    mock_construct_annotated_pages: Mock,
+    mock_construct_annotated_doc: Mock,
+    mock_update_task_status: Mock,
+    mock_logger_exception: Mock,
```
Comment (cakeinsauce) [inline annotation/tests/test_tasks_services.py:1757]: fix typings, please. not all of your fixtures return mocks

File: annotation/tests/test_tasks_services.py [hunk 2]
```diff
@@ -1746,3 +1746,120 @@ def test_get_tasks_without_ids_with_multiple_task_types():
...
+    mock_logger_exception.assert_not_called()
+
+
+def test_create_validation_revisions_multiple_tasks(
+    mock_session: Mock,
+    mock_get_file_path_and_bucket: Mock,
+    mock_construct_annotated_pages: Mock,
+    mock_construct_annotated_doc: Mock,
+    mock_update_task_status: Mock,
+    mock_logger_exception: Mock,
```
Comment (cakeinsauce) [inline annotation/tests/test_tasks_services.py:1781]: fix typings, please. not all of your fixtures return mocks

File: annotation/tests/test_tasks_services.py [hunk 3]
```diff
@@ -1746,3 +1746,120 @@ def test_get_tasks_without_ids_with_multiple_task_types():
...
+        mock_logger_exception.assert_not_called()
+
+
+def test_create_validation_revisions_multiple_tasks_value_error(
+    mock_session: Mock,
+    mock_get_file_path_and_bucket: Mock,
+    mock_construct_annotated_pages: Mock,
+    mock_construct_annotated_doc: Mock,
+    mock_update_task_status: Mock,
+    mock_logger_exception: Mock,
```
Comment (cakeinsauce) [inline annotation/tests/test_tasks_services.py:1825]: fix typings, please. not all of your fixtures return mocks

File: annotation/tests/test_tasks_services.py (hunk 1 above)
Comment (MilanBogicevic8) [reply annotation/tests/test_tasks_services.py:1757]: This issue is from before the test changes were made.

File: annotation/tests/test_tasks_services.py (hunk 2 above)
Comment (MilanBogicevic8) [reply annotation/tests/test_tasks_services.py:1781]: This issue is from before the test changes were made.

File: annotation/tests/test_tasks_services.py (hunk 3 above)
Comment (MilanBogicevic8) [reply annotation/tests/test_tasks_services.py:1825]: This issue is from before the test changes were made.

**Pull request merged**